slackclient/_server.py

```diff
@@ -105,8 +107,8 @@ def connect_slack_websocket(self, ws_url):
                                                http_proxy_port=proxy_port,
                                                http_proxy_auth=proxy_auth)
             self.websocket.sock.setblocking(0)
-        except Exception as e:
-            raise SlackConnectionError(str(e))
+        except (websocket.WebSocketException, socket.error):
+            raise SlackConnectionError
```

Comment:
everything will look like a `SlackConnectionError` with no details from the actual exception?

Reply:
Ya, I'd at least like the message or errno (that should be in `socket.error` .... or both.